pkg/defaults/defaults.go

```diff
@@ -68,6 +68,7 @@ func FromConfig(
 	var rbacClient rbacclientset.RbacV1Interface
 	var saGetter coreclientset.ServiceAccountsGetter
 	var namespaceClient coreclientset.NamespaceInterface
+	var eventClient coreclientset.EventsGetter
```

Comment:
instead of adding a 13th typed client, can we add the controller-runtime client and try to replace over time the existing typed clients with it? That would greatly simplify the code and testability

Reply:
Orthogonal to this effort, please.

Reply:
Replacing existing client, yes. Not adding a new typed client vs using the dynamic one, no.

Reply:
Just another PR please.